Kitodo/src/main/java/de/sub/goobi/config/Parameters.java

```diff
@@ -0,0 +1,695 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package de.sub.goobi.config;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * These constants define configuration parameters usable in the configuration
+ * file. This file reflects the order of the global
+ * {@code kitodo_config.properties} file (as far as possible), and vice versa.
+ */
+public class Parameters {
+
+    // =============================================================================
+    // FILE AND DIRECTORY MANAGEMENT
+    // =============================================================================
+    // -----------------------------------
+    // Directories
+    // -----------------------------------
+
+    // DIR_XML_CONFIG = "KonfigurationVerzeichnis" → org.kitodo.config.Config
```

Comment:
Why this comment?

Reply:
`Parameters.java` has been sorted to corresponding to `kitodo_config.properties`, including the section headers as you see. However there is an entry `KonfigurationVerzeichnis` in  `kitodo_config.properties`, but there is no constant for it here. This comment is to explain where the constant is if someone compares the two files in the future.